OutOfSchool/OutOfSchool.BusinessLogic/Models/WorkshopDraft/WorkshopDraftRejectionDto.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace OutOfSchool.BusinessLogic.Models.WorkshopDraft;
+public class WorkshopDraftRejectionDto
+{    
+    public string RejectionMessage { get; set; }
```

Comment:
@annygumeniuk  I agree with rabbit. We need to set max length for message. Other rejection messages in blocking, etc. Have the length of `500`